commands/daemonize/daemonize.go

```diff
@@ -0,0 +1,70 @@
+package daemonize
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/takama/daemon"
+	cmd_ep "github.com/taskcluster/taskcluster-worker/commands/extpoints"
+	"github.com/taskcluster/taskcluster-worker/runtime"
+)
+
+const (
+	// name of the service
+	name        = "taskcluster-worker"
+	description = "Taskcluster worker"
+)
+
+var dependencies = []string{}
+
+func init() {
+	cmd_ep.CommandProviders.Register(cmd{}, "daemon")
+}
+
+type cmd struct{}
+
+func (cmd) Summary() string {
+	return "Run as a daemon."
+}
+
+func usage() string {
+	return `Usage:
+  taskcluster-worker daemon <action> [--engine <engine>] [--logging-level <level>]
+
+Options:
+  -l --logging-level <level>  	Set logging at <level>.
+  -e --engine <engine>			Set the engine at <engine>.
+`
+}
+
+func (cmd) Usage() string {
+	return usage()
+}
```

Comment:
it is used in service.go file when an invalid command is specified, although we can `panic`